source/sonata_simulation.rst

```diff
@@ -96,6 +97,27 @@ example::
        "integration_method" : 2
   }
 
+Format of the electrodes_file
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The format of the weights file is described below:
+
+.. table::
+
+  ============================= ======================= ========== =============== ============= ==========================================
+  Group                         Dataset                 Type       Shape           Requirement   Description
+  ============================= ======================= ========== =============== ============= ==========================================
+  /electrodes/{electrodename}   layer                   utf8       1               Optional      Layer of the circuit in which {electrodename} is located
+  /electrodes/{electrodename}   position                float32    3               Optional      Position of {electrodename} in microns, in cartesian coordinates (3 x 1)
+  /electrodes/{electrodename}   region                  utf8       1               Optional      Region in which {electrodename} is located
+  /electrodes/{electrodename}   type                    utf8       1               Optional      Either EEG or LFP
+  /electrodes/{electrodename}   offset                  uint64     1               Optional      Column index of the coefficients for {electrodename} in /electrodes/{population_name}/{node_id}
```

Comment:
since this is optional, what happens if it doesn't exist?

Reply:
I'm not sure what it's meant to represent, and if it's not used, I'm not sure we should add it, since it distracts from understanding the format.

Reply:
@jorblancoa are you ok with not including it?

Reply:
yes, we shouldnt need it since all {electrodename} share the node_ids and sec_ids